catalog/overture-maps/overture-maps_places_place.jsonnet

```diff
@@ -0,0 +1,200 @@
+    effort to develop interoperable open map data. The data is released
+    monthly and always reflects the most recent release.
+
+    Data source: [Overture Maps Foundation](https://overturemaps.org/)
+
+    Documentation: [Places Theme Guide](https://docs.overturemaps.org/guides/places/)
+
+    BigQuery table: `bigquery-public-data.overture_maps.place`
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference),
+    {
+      rel: ee_const.rel.source,
+      href: 'https://console.cloud.google.com/bigquery?p=bigquery-public-data&d=overture_maps&t=place',
+    },
+  ],
+  'gee:categories': [
+    'infrastructure-boundaries',
+  ],
+  keywords: [
+    'global',
+    'map',
+    'population',
+    'urban',
+  ],
+  providers: [
+    ee.producer_provider(
+      'Overture Maps Foundation',
+      'https://overturemaps.org/'
+    ),
+    ee.processor_provider(
+      'CARTO',
+      'https://carto.com/'
+    ),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2023-07-01T00:00:00Z', null),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'addresses',
+        description: 'The addresses of the place. RECORD type.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'bbox',
+        description: |||
+          Bounding box defined by two longitudes and two latitudes.
+          RECORD type with fields: xmin, xmax, ymin, ymax.
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'brand',
+        description: |||
+          The brand of the place. Locations with multiple brands are
+          considered as separate places. RECORD type.
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'categories',
+        description: |||
+          The categories of the place. Complete list available on GitHub.
+          RECORD type with primary and alternate categories.
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'confidence',
+        description: |||
+          The confidence of the existence of the place. A number between 0
+          and 1, where higher values indicate greater certainty.
+        |||,
+        type: ee_const.var_type.double,
+      },
+      {
+        name: 'emails',
+        description: 'The email addresses of the place. RECORD type.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'geometry',
+        description: 'Point geometry in WGS84 (EPSG:4326).',
+        type: ee_const.var_type.geometry,
+      },
+      {
+        name: 'id',
+        description: |||
+          A feature ID. This may be an ID associated with the Global Entity
+          Reference System (GERS).
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'names',
+        description: |||
+          The names used when referring to a feature. The first entry must
```

Comment:
Looks like "names" is a single object - how can there be multiple?

Reply:
In the documentation appears that that field includes: Primary (main name), Common (with common translations), and Rules (Rules for names that cannot be specified in the simple common names property. These rules can cover other name variants such as official, alternate, and short; and they can optionally include geometric scoping (linear referencing) and side-of-road scoping for complex cases.)
Reference: https://docs.overturemaps.org/schema/reference/places/place/